xtra-libp2p-offer/src/taker.rs

```diff
@@ -28,7 +27,7 @@ impl Actor {
             let offers = protocol::recv(stream).await?;
             let span = tracing::debug_span!("Received new offers from maker", %peer, ?offers);
             maker_offers
-                .send(LatestMakerOffers(offers))
+                .send(LatestMakerOffers(offers.map(|offer| offer.into())))
```

Comment:
🙃 I prefer this if it works.

```suggestion
                .send(LatestMakerOffers(offers.map(model::MakerOffers::from)))
```